raui-core/src/props.rs

```diff
@@ -326,6 +328,8 @@ impl From<&Self> for Props {
     }
 }
 
+/// Macro for implementing [`PropsData`]. You may prefer to use the [derive
+/// macro][`macro@crate::PropsData`] instead.
```

Comment:
add: "but in case of auto-implementing PropsData and Prefab traits for remote or std types, you might find this macro useful"